docs/CONFIG-VARS.md

```diff
@@ -408,7 +408,7 @@ The EBS CSI driver is currently only used for kubernetes v1.23 or later AWS EKS
 | INGRESS_NGINX_NAMESPACE | NGINX Ingress Helm installation namespace | string | ingress-nginx | false | | baseline |
 | INGRESS_NGINX_CHART_URL | NGINX Ingress Helm chart URL | string | See [this document](https://kubernetes.github.io/ingress-nginx) for more information. | false | | baseline |
 | INGRESS_NGINX_CHART_NAME | NGINX Ingress Helm chart name | string | ingress-nginx | false | | baseline |
-| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version 4.3.0 is used for Kubernetes clusters whose version is <= 1.23.X, and version 4.7.1 is used for Kubernetes clusters whose version is >= 1.24.X. | baseline |
+| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version 4.3.0 is used for Kubernetes clusters whose version is <= 1.24.X, and version 4.9.0 is used for Kubernetes clusters whose version is >= 1.25.X. | baseline |
```

Comment:
In the last release we stated that we needed 4.7.1 for 1.24.x and great. But now we're saying for anything less than and up to 1.24.x we use 4.3.0? Seems off

Reply:
I know it is little odd. Unfortunately, AWS has an extended support for 1.23 until October 11, 2024 and extended support for 1.24 until January 31, 2025. 
The version `4.7.1` does not support 1.23, and the newer version doesn't support 1.24. So had to do this kind of mix, as `4.3.0` supports the range `1.25, 1.24, 1.23, 1.22`.

Reply:
ok so seems we need a table or range because we can't tell folks that 4.3.0 now magically works for 1.24.x now. Where we explicitly stated it did not.

Reply:
I can understand the LTS suport implications for the SAS Viya software. Accepting this change.